tests/test_all_fields.py

```diff
@@ -165,8 +162,7 @@
     'invoices': {'updated'},
     'plans': {'updated'},
     'invoice_line_items': {
-        'updated',
```

Comment:
Tap does write the `updated` field in `invoice_line_items` as it has no replication key value in the object itself available. `invoice_line_items` is bookmarked based on parent invoices. [Reference](https://github.com/singer-io/tap-stripe/blob/master/tap_stripe/__init__.py#L62)